docs/source/en/model_doc/modernbert-decoder.md

```diff
@@ -65,9 +63,9 @@ classifier("This movie is really great!")
 import torch
 from transformers import AutoModelForCausalLM, AutoTokenizer
 
-tokenizer = AutoTokenizer.from_pretrained("blab-jhu/test-32m-dec")
+tokenizer = AutoTokenizer.from_pretrained("jhu-clsp/ettin-decoder-17m")
 model = AutoModelForCausalLM.from_pretrained(
-    "blab-jhu/test-32m-dec",
+    "jhu-clsp/ettin-decoder-17m",
```

Comment:
For the `AutoModel` examples, maybe add `attn_implementation=flash_attention_2" to demonstrate FlashAttention support